optics/CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# optics-0.4 (2021-mm-dd)
+* See [migration-guide-0.4.md](https://github.com/well-typed/optics/blob/master/migration-guide-0.4.md) for more details
+* Add support for GHC-9.0
+* Drop support for GHC-8.0
+* The `FunctorWithIndex`, `FoldableWithIndex` and `TraversableWithIndex` type
+  classes have been migrated to a new package,
+  [`indexed-traversable`](https://hackage.haskell.org/package/indexed-traversable)
+* Add `adjoin`, `iadjoin` and `both` to `Optics.Traversal`
+* Generalize types of `generic`
+* Make `chosen` an indexed lens to see which value is traversed
+* Add `gfield`, `gafield`, `gconstructor`, `gposition` and `gplate` for
+  generics-based data access
+* Add support for generics-based field lenses and constructor prisms (`gfield`
+  and `gconstructor`) to `LabelOptic` so they can be used via `OverloadedLabels`
+* Remove unnecessary INLINE pragmas to reduce compile times (#394)
+* Simplify the type of `(%)` using new `JoinKinds` and `AppendIndices` classes
+  in place of the `Join` and `Append` type families
+* Print missing language extensions during TH generation of labels if there are
+  any
+* Add support for getters of rank1 polymorphic fields to optics generated with
+  the `makeFieldLabels` family of functions
+* Extend support of type-changing optics generated with the `makeFieldLabels`
+  family to type parameters that are phantom and applied to non-injective type
+  families
+* Fix TH generation of optics for poly-kinded data families (#378)
+* Fix `declareFieldLabels` when a field type refers to a type defined in the
+  same quote (#380)
```

Comment:
Note, these aren't magically turned into links. Should they be?

Reply:
I was being lazy, but now I've done it the brute force way.